components/support/rc_crypto/src/ece.rs

```diff
@@ -0,0 +1,491 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::{
+    aead,
+    agreement::{self, Curve, EcKey},
+    digest, hkdf, hmac, rand,
+};
+use ece::{
+    Aes128GcmEceWebPush, AesGcmEceWebPush, Crypto, ErrorKind, Result,
+    ECE_WEBPUSH_AUTH_SECRET_LENGTH,
+};
+pub use ece::{
+    AesGcmEncryptedBlock, EcKeyComponents, Error, LocalKeyPair, RemotePublicKey, WebPushParams,
+};
+
+// Can't `impl From<crate::Result<T>> for ece::Result<T>`
+// because we don't own either struct :-(
```

Comment:
Is this comment still true now that it's part of the `rc_crypto` crate?

Reply:
Yup still true :/